Toolkit/ArcGISToolkit/UnitsViewController.swift

```diff
@@ -61,23 +61,12 @@ public class UnitsViewController: TableViewController {
     /// The search controller used by the units view controller. 
     private var searchController: UISearchController? {
```

Comment:
This property is no longer necessary. I would suggest eliminating it and replacing all uses of it with `navigationItem.searchController`.

Reply:
good call